### PR TITLE
Fix issue with PubNub logs channel. If available, use device.logs_cha…

### PR DESCRIPTION
**Pull request by nghiant2710**
…nnel for PubNub channel names

### CODE REVIEW

File: resin/logs.py [hunk 1]
```diff
@@ -105,4 +106,12 @@ def get_channel(uuid):
 
         """
 
-        return 'device-{uuid}-logs'.format(uuid=uuid)
+        if not hasattr(self, 'logs_channel'):
+            device_info = self.device.get(uuid)
+            print device_info
```
Comment (dfunckt) [inline resin/logs.py]: :)

File: resin/logs.py [hunk 2]
```diff
@@ -105,4 +106,12 @@ def get_channel(uuid):
...
+            if 'logs_channel' in device_info:
+                self.logs_channel = device_info['logs_channel']
+            else:
+                self.logs_channel = uuid
+
+        return 'device-{logs_channel}-logs'.format(logs_channel=self.logs_channel)
```
Comment (dfunckt) [inline resin/logs.py:116]: This need not be so verbose--it's a single variable. Maybe use the {0} shorthand?

**Comment by dfunckt** [comment]
LGTM

File: resin/logs.py (hunk 1 above)
Comment (nghiant2710) [inline resin/logs.py]: removed :D

File: resin/logs.py (hunk 2 above)
Comment (nghiant2710) [inline resin/logs.py:116]: it will give me better context when lookat in the future

**Pull request merged**